manifests/integreatly-codeready-workspaces/2.2.0/crwoperator.v2.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,500 @@
+    name: plugin-java11-rhel8
+  - image: registry.redhat.io/codeready-workspaces/plugin-kubernetes-rhel8@sha256:18c64ed3cbc607d69f42baa7f4b8a5064d55b432d0a7bafc6e7815f01ade78c7
+    name: plugin-kubernetes-rhel8
+  - image: registry.redhat.io/codeready-workspaces/plugin-openshift-rhel8@sha256:fa05da28a3324abe0778ad35a9fd6ca7f19e185ad4be9e2bedc8d2ac6315c421
+    name: plugin-openshift-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-cpp-rhel8@sha256:48620ca9b2d9badd00ecb71b2080bac41e477c84b7a4211a2473cea529fd8c52
+    name: stacks-cpp-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-dotnet-rhel8@sha256:341332f429b915e42e1f136be230ec9b77308e401027ec45794a636b056b795f
+    name: stacks-dotnet-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-golang-rhel8@sha256:d27d8e2550ce6a5867dff65754a3ea88ffab350d7e08dbb49ad0a85573c5b3f8
+    name: stacks-golang-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-java-rhel8@sha256:573ca673f972ba6a73d21da03cef927f753604d7a4e452f44ea6ee76d9d03394
+    name: stacks-java-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-node-rhel8@sha256:09f867835d372a91a5390c083decc98854efd9e2040ec2f89599e026f16a1ee7
+    name: stacks-node-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-php-rhel8@sha256:62ffa0e5e320084809716370fba5c657c40638cf35b83687ec2e38fdd0630042
+    name: stacks-php-rhel8
+  - image: registry.redhat.io/codeready-workspaces/stacks-python-rhel8@sha256:a83ff1752b56003c687a6ad4c110ac86a7f7062e107a61f668e0413206913537
+    name: stacks-python-rhel8
+  - image: registry.redhat.io/codeready-workspaces/theia-endpoint-rhel8@sha256:68472d3b771111c8bf8139a773dcf50f1b276aa3a4186814dea2eb5ae7fd4fc9
+    name: theia-endpoint-rhel8
+  - image: registry.redhat.io/codeready-workspaces/theia-rhel8@sha256:9438b3ec8ac2fc1d9cfff699d40059b5a01d782fd4cb26f30efcbf6407c82327
+    name: theia-rhel8
+  - image: registry.redhat.io/ubi8/openjdk-11@sha256:a9b1a21661411b692cb621b5f1dc55e8ac6324e5f342c3fa80e85c315975373b
+    name: openjdk-11
+  - image: registry.redhat.io/ubi8/openjdk-8@sha256:573fdab1e0d40fdb5b786c13c50be542258fe65fc22904d3ab1588389330acc7
+    name: openjdk-8
+  - image: registry-proxy.engineering.redhat.com/rh-osbs/jboss-eap-7-eap-xp1-openjdk11-openshift-rhel8@sha256:94e1cd4eb4196a358e301c1992663258c0016c80247f507fd1c39cf9a73da833
+    name: jboss-eap-7-eap-xp1-openjdk11-openshift-rhel8
+  - image: registry-proxy.engineering.redhat.com/rh-osbs/jboss-eap-7-eap-xp1-openjdk8-openshift-rhel7@sha256:973c66d587b45e1d8985e769c11e1f5b93eb04803a61471a66a6f2f847499dd3
+    name: jboss-eap-7-eap-xp1-openjdk8-openshift-rhel7
+  - image: registry.redhat.io/rhscl/mongodb-36-rhel7@sha256:9f799d356d7d2e442bde9d401b720600fd9059a3d8eefea6f3b2ffa721c0dc73
+    name: mongodb-36-rhel7
+  replaces: crwoperator.v2.0.0
```

Comment:
Can we really skip 2.1 minor version? 
Would be nice to confirm in the docs or with the CRW team.

If we are going to skip 2.1.x, then we can close #765